src/components/BuildBaronAndAnnotations/BuildBaronContent.tsx

```diff
@@ -100,17 +100,20 @@ const BuildBaronCore: React.FC<BuildBaronCoreProps> = ({
               />
             </>
           )}
-
```

Comment:
Right now the File Ticket Button is only configured for teams who use the build baron (to be configured for other teams in other tickets). We don't want someone from other teams to attempt to use it. Since we will now show this tab to them, we should disable that button [here](https://github.com/evergreen-ci/spruce/blob/master/src/components/BuildBaronAndAnnotations/BBFIleTicket.tsx#L82) with a tooltip giving them some information.

Reply:
![image](https://user-images.githubusercontent.com/26798134/104633790-91e6dd80-5654-11eb-91b5-41cc88c8ae32.png)
I think the tooltip might be confusing, since this will be eventually folded into task annotations. I think just hiding it for the moment will do the trick.